src/components/Schedule/ProgressBar.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from "react";
+
+interface IProps {
+  actual: number;
+  max: number;
+}
+
+export const ProgressBar: FC<IProps> = ({ actual, max }) => (
+  <div className="pt-8 border-b border-solid border-lilac relative flex items-center rounded-12">
```

Comment:
don't create the bar with borders but backgrounds, and a specific height on the inner div
```
<div className="mt-8 bg-lilac rounded-12">
  {actual !== 0 && (
    <div
      style={{ width: (actual / max) * 100 + "%", height: 3px }}
      className="bg-black rounded-12"
    />
  )}
</div>
```